### PR TITLE
Telegram notification support

### PR DESCRIPTION
**Pull request by Philenst**
Adds support for sending to Telegram:

image

Introduces 2 new configs:
- bot_token, which should be a bot token
- chat_id, which should be the chat ID of the user receiving the notifications.

Does require to duplicate some logs however not really avoidable without writing dedicated functions for it.

Also splits off Send-Payload to Send-JSONPayload (As it converts the body to JSON) and has a new Send-Payload function.

Ideally replaces PR #49

### CODE REVIEW

File: AlertSender.ps1 [hunk 1]
```diff
@@ -352,37 +352,64 @@ try {
...
+							Write-LogMessage -Tag 'DEBUG' -Message 'Setting user name for mention.'
+							$payloadParams.UserName = $service.Value.user_name
+						}
+					}
 
-			If ($service.Value.webhook.StartsWith('https')) {
-				Write-LogMessage -Tag 'INFO' -Message "Sending notification to $($serviceName)."
-				$logId_service = $vbrSessionLogger.AddLog("[VeeamNotify] Sending notification to $($serviceName)...")
+					# Get URI from webhook value
+					$uri = $service.Value.webhook
```
Comment (tigattack) [inline AlertSender.ps1:375]: What's the reason for these changes?

**Review by tigattack — CHANGES_REQUESTED** [review]
Most of this looks pretty good, thanks Phil.

Couple questions and suggestions, but I'm happy to merge once resolved.

File: AlertSender.ps1 (hunk 1 above)
Comment (tigattack) [reply AlertSender.ps1:375]: I selected the wrong line numbers for this comment but I'm referring to everything except the first and last change here:
image

File: resources/NotificationHandler.psm1 [hunk 2]
```diff
@@ -679,7 +682,109 @@ function New-SlackPayload {
...
+# 	Method      = 'Post'
+# 	ContentType = 'application/x-www-form-urlencoded'
+# 	ErrorAction = 'Continue'
+# }
+
+# # Post payload
+# [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
+# Invoke-RestMethod @postParams
+
+function Send-JSONPayload {
```
[the remark this answers is not in the document]
Comment (Philenst) [reply resources/NotificationHandler.psm1]: Should be solved by https://github.com/tigattack/VeeamNotify/pull/63/commits/bec6452397a50b423629e6485a8d87f9882405a0

File: AlertSender.ps1 (hunk 1 above)
Comment (Philenst) [reply AlertSender.ps1:375]: Are you referring to the $Uri being moved or reassigned? Sorry, not sure what you mean.

Comment (tigattack) [reply AlertSender.ps1:375]: Sorry, I forget exactly what I was querying now, but I'd guess that's what I meant. We can leave this since it's not a big deal, not sure what I had in mind at the time.

**Review by tigattack — APPROVED** [review]
Thanks Phil ❤️
